client/src/components/Card.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+import PropTypes from 'prop-types';
+import Input from './Input';
+
+const CardContainer = styled.div`
+  width: 300px;
+  height: 430px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid ${colors.complementaryPrimary};
+  border-radius: 24px;
+  background-color: ${colors.backgroundPrimary};
+  font-size: 20px;
+  color: ${colors.fontPrimary};
+  box-shadow: 10px 15px 10px ${colors.backgroundPrimary};
+`;
+
+const CardTitleContainer = styled.div`
+  width: 274px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+  padding: 10px;
+  font-size: 1.5rem;
+  border-bottom: 1px solid ${colors.complementaryPrimary};
+`;
+
+const CardDataContainer = styled.div`
+  width: 234px;
+`;
+
+const CardDataPriceContainer = styled.div`
```

Comment:
these containers look pretty similar. maybe you can make like a general flexbox container styled component? with shared props and then if necessary style it when you need additional props like align-items etc. and reuse it below? 
of course you would lose some of the semantic meaning. then you could do something like have  `const CardDataPriceContainer =  styled(flexboxcontainer)`
do you know what I mean? 😅

Reply:
I combine some of these containers in my latest commit. Good hint!